pyomo/util/infeasible.py

```diff
@@ -37,6 +37,13 @@ def log_infeasible_constraints(
         log_variables (bool): If true, prints the constraint variable names and values
 
     """
+    if logger.getEffectiveLevel() > logging.INFO:
+        logger.warning(
+            'log_infeasible_constraints() called with a logger whose '
+            'effective level is higher than logging.INFO: no output '
+            'will be logged reguardless of constraint feasibility'
```

Comment:
Repeated Typo: regardless

Reply:
Should this warning include a note on how to change the logging level?
"This warning can be avoided by changing the Pyomo logging level: `<CODE TO CHANGE LOGGING LEVEL>`"

Reply:
I don't think so.  There are too many ways to change the logging level (adding handlers, calling a standard setup, setting the logging level) to put something specific here.  If we start to see a bunch of questions about it, then we can add a reference to the online docs (in the [Common Warnings/Errors](https://pyomo.readthedocs.io/en/latest/errors.html#warnings) list) -- where we can have a longer and more detailed discussion.